modules/aggregations/build.gradle

```diff
@@ -27,6 +28,15 @@ restResources {
   }
 }
 
+if (BuildParams.isSnapshotBuild() == false) {
+  tasks.named("test").configure {
+    systemProperty 'es.index_mode_feature_flag_registered', 'true'
```

Comment:
There's a nice thing for this now!

```
  requiresFeature 'es.index_mode_feature_flag_registered', Version.fromString("8.0.0")
```

Reply:
I've merged this PR now, so that the release tests stop failing.
If `requiresFeature` can also be used here then I can change that in a follow up PR.